esmvaltool/diag_scripts/seaice/seaice_sensitivity.py

```diff
@@ -0,0 +1,343 @@
+"""Diagnostic that shows the sensitivity of sea ice area to global warming."""
+
+import logging
+from pathlib import Path
+
+import iris
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.colors import Normalize
+from scipy import stats
+
+from esmvaltool.diag_scripts.shared import (
+    run_diagnostic,
+    save_figure,
+    select_metadata,
+)
+
+logger = logging.getLogger(Path(__file__).stem)
+
+
+# This is stolen from the example in AutoAssess _plot_mo_metrics.py
+def get_provenance_record(cfg):
+    """Create a provenance record describing the diagnostic data and plot."""
+    filenames = [item["filename"] for item in cfg["input_data"].values()]
+
+    region = [item["diagnostic"] for item in cfg["input_data"].values()][0]
+
+    record = {
+        "caption": f"Plots of {region.title()} sea ice sensitivity",
```

Comment:
From discussion at ESMValTool workshop today there is a request for all REF-focused recipes to ensure that provenance captions are meaningful and specific, because the captions will be used in web publication of plots.

We therefore probably should have a different caption for each plot. I suggest that we add a `caption` argument to this function to allow the calling routines to specify it themselves. I recommend using the fig_title variables (without the ax_title subheadings.